misc/96rapiddisk/module-setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+	return 0
+}
+
+depends() {
+	echo dm
+}
+
+installkernel() {
+
```

Comment:
(style) no big deal and you do not have to but to maintain consistency with other functions, I would remove the blank lines above and below the "hostonly=" line.